### PR TITLE
翻译docs/cn/advanced-concepts/settings.md

### PR DESCRIPTION
**Pull request by ghost62184**
## Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce to Appium?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I have read the [CONTRIBUTING](/CONTRIBUTING.md) doc
- [ ] I have signed the [CLA](https://docs.google.com/forms/d/1lOfXRw_0VCk7gYzjj4WLetGu7yelDVo5LWh0z7pGftE/viewform)
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: docs/cn/advanced-concepts/settings.md [hunk 1]
```diff
@@ -1,23 +1,22 @@
...
+Settings是appium引入的一个新概念。 它们目前不是Mobile JSON Wire协议或Webdriver规范的一部分。
 
-Settings are a new concept introduced by appium. They are currently not a part of the Mobile JSON Wire Protocol, or the Webdriver spec.
+Settings是用来指定appium server的工作方式。
 
-Settings are a way to specify the behavior of the appium server.
+Settings有以下特点:
+ - 可变性，Settings在一个会话中是可以被修改的。
+ - 临时性，Settings只对当前会话生效，新建立的会话会被重置。
+ - 局限性，Settings只能控制appium server，不能控制被测应用或设备。
```
Comment (yanqian) [inline docs/cn/advanced-concepts/settings.md]: 「不能」感觉有点生硬，修改成「不被用于」是不是更好一点？

File: docs/cn/advanced-concepts/settings.md [hunk 2]
```diff
@@ -1,23 +1,22 @@
...
- - Control the way the appium server behaves during test automation. They do not apply to controlling the app or device under test.
+以Android的`ignoreUnimportantViews`为例。Android中可以设置`ignoreUnimportantViews`用来忽略所有与当前视图无关的元素，这样可以让用例执行的更快。注意，当用户需要访问这些被忽略的元素时，需要禁用`ignoreUnimportantViews`后并重新启用。 
 
-An example of a setting would be `ignoreUnimportantViews` for Android. Android can be set to ignore elements in the View Hierarchy which it deems irrelevant. Setting this can cause tests to run faster. A user who *wants* to access the ignored elements however, would want to disable `ignoreUnimportantViews`, and reenable it afterwards.
+另外一个例子是Settings让appium忽略当前不可见的元素。
 
-Another example of a use-case for settings would be telling appium to ignore elements which are not visible.
-
-Settings are implemented via the following API endpoints:
+Settings可以通过下面的API实现：
```
Comment (yanqian) [inline docs/cn/advanced-concepts/settings.md]: 「可以通过下面的API实现」可能会引起误解，以为还可以通过其他API实现。建议换成，「Settings是通过下面的API实现」

File: docs/cn/advanced-concepts/settings.md [hunk 3]
```diff
@@ -1,23 +1,22 @@
...
 
 **POST** /session/:sessionId/appium/settings
 
->Expects a JSON hash of settings, where keys correspond to setting names, and values to the value of the setting.
+>以JSON格式进行配置。
```
Comment (yanqian) [inline docs/cn/advanced-concepts/settings.md]: 这样翻译有点粗略啦

File: docs/cn/advanced-concepts/settings.md [hunk 4]
```diff
@@ -28,29 +27,29 @@ Settings are implemented via the following API endpoints:
...
->Returns a JSON hash of all the currently specified settings.
+>以JSON格式返回当前所有配置。
 ```
 {
   ignoreUnimportantViews : true
 }
 ```
 
-Note that the actual commands you would use in your test script differ based on the language; see the specific Appium client documention for more information.
+注意，实际的命令因您的脚本所使用的语言而异; 请参阅具体的Appium客户端文档以获取更多信息。
```
Comment (yanqian) [inline docs/cn/advanced-concepts/settings.md]: 赞！！

File: docs/cn/advanced-concepts/settings.md [hunk 5]
```diff
@@ -28,29 +27,29 @@ Settings are implemented via the following API endpoints:
...
 
-### Supported Settings
+### 支持的Settings
 
-**"ignoreUnimportantViews"** - Boolean which sets whether Android devices should use `setCompressedLayoutHeirarchy()` which ignores all views which are marked IMPORTANT_FOR_ACCESSIBILITY_NO or IMPORTANT_FOR_ACCESSIBILITY_AUTO (and have been deemed not important by the system), in an attempt to make things less confusing or faster.
+**"ignoreUnimportantViews"** - 布尔类型，被设置为true时，Android设备会使用`setCompressedLayoutHeirarchy()`忽略标记了IMPORTANT_FOR_ACCESSIBILITY_NO或IMPORTANT_FOR_ACCESSIBILITY_AUTO（以及被系统认为不重要的）的元素，这样可以让脚本变得简单或执行的更快。
```
Comment (yanqian) [inline docs/cn/advanced-concepts/settings.md]: 这句话读英文可以理解，但是翻译就很难了，很厉害。一些小的建议：
- 「元素」翻译成「视图」是不是好点？
- 「in an attempt」我觉得应该是在说，尝试但不一定会达到目的，所以，翻译成「可以」稍微有点不恰当。
- 你说了设置为「TRUE」的情况，稍微提一下「False」的情况会更完整一些吧。

File: docs/cn/advanced-concepts/settings.md (hunk 2 above)
Comment (ghost62184) [inline docs/cn/advanced-concepts/settings.md]: 恩，这个地方翻译的有问题，谢谢指正

File: docs/cn/advanced-concepts/settings.md (hunk 3 above)
Comment (ghost62184) [inline docs/cn/advanced-concepts/settings.md]: 我再完善一下

File: docs/cn/advanced-concepts/settings.md (hunk 5 above)
Comment (ghost62184) [inline docs/cn/advanced-concepts/settings.md]: 恩，细节翻译的不好，我再改改，多谢多谢

**Comment by yanqian** [comment]
lgtm @ghost62184

**Pull request merged**